frontend/ui/widget/opdsbrowser.lua

```diff
@@ -58,7 +58,37 @@ local OPDSBrowser = Menu:extend{
 }
 
 function OPDSBrowser:init()
-    self.item_table = self:genItemTableFromRoot(self.opds_servers)
+    local servers = G_reader_settings:readSetting("opds_servers") or {}
+    if next(servers) == nil then -- If there are no saved servers, add some defaults
+        servers = {
+          {
+	     title = "Project Gutenberg",
```

Comment:
I just got my editor sorted out and there shouldn't be any more errant tabs in the future.